js-apps/meep-frontend/src/js/util/scenario-utils.js

```diff
@@ -1125,6 +1149,66 @@ export function createPoaCell(uniqueId, name, element) {
   return nl;
 }
 
+export function createPoa5G(uniqueId, name, element) {
```

Comment:
not sure what to gain from putting all in one bigger function (the part of reusable code is minimal), i suggest keeping them separated rather than ending with a big switch/case.